internal/kubernetes/kubernetes.go

```diff
@@ -122,8 +123,15 @@ func (c Cluster) Kubectl() Control {
 }
 
 func (c Cluster) isAvailable(ctx context.Context) error {
-	_, err := c.Kubectl().Run(ctx, "api-versions")
-	return err
+	out, err := c.Kubectl().Run(ctx, "api-versions")
+	if err != nil {
+		return err
+	}
+	if len(strings.TrimSpace(out)) == 0 {
+		return fmt.Errorf("no api versions?")
+	}
```

Comment:
Not related to this change, but I found that `kubectl api-versions` succeeds with an empty output if there is no context configured. I can move this to a different PR if desired.